app/templates/index.html

```diff
@@ -7,7 +7,7 @@
   <div class="text-center"> 
     <h1> {{ title }}</h1>
     <h3>Filter and cluster compounds</h3> 
-    <p> Enter a csv file containing three columns labeled 'smile', 'label', and 'mpo'.</p> 
+    <p> Enter a .csv file with columns titled SMILES and label.</p> 
```

Comment:
Can you add 'SMILES' to smiles.py when we read in the data? I know you can't run it, but i want to make sure we're consistent. (I can test run for you) It would also be great if you could update the example files